### ISSUE TITLE
Allow passing include paths to windres/rc

### ISSUE THREAD

**Issue by roblabla** [nabijaczleweli/rust-embed-resource#32]
windres, rc and llvm-rc (through the C preprocessor) all have the ability to pass additional include paths to be used by the preprocessor, e.g. `windres -I path` or `rc /I path`. Exposing this to the command line would be useful to allow users to generate header files that they could then use in their `resource.rc`. 

This would allow some more sophisticated resource files to get embedded. Here, for instance, we generate a `cargo.h` that contains defines for various environment variables cargo gives us, and an RC file that includes it and uses it to generate a VS_VERSION_INFO section with the correct version.

- <details><summary>RC file</summary>

  ```c
  #include <cargo.h>
  #define VER CARGO_PKG_VERSION_MAJOR,CARGO_PKG_VERSION_MINOR,CARGO_PKG_VERSION_PATCH,0
  VS_VERSION_INFO VERSIONINFO
  FILEVERSION VER
  PRODUCTVERSION VER
  BEGIN
  END
  ```

  </summary>
- <details><summary>build.rs files</summary>

  ```rust
  fn main() {
      // Generate cargo.h, included by resources.rc
      let out_dir = PathBuf::from(std::env::var_os("OUT_DIR").unwrap());
      std::fs::write(out_dir.join("cargo.h"), format!(r#"
  #define CARGO_PKG_VERSION "{}"
  #define CARGO_PKG_VERSION_MAJOR {}
  #define CARGO_PKG_VERSION_MINOR {}
  #define CARGO_PKG_VERSION_PATCH {}
  #define CARGO_PKG_VERSION_PRE {}"#,
          std::env::var("CARGO_PKG_VERSION").unwrap(),
          std::env::var("CARGO_PKG_VERSION_MAJOR").unwrap(),
          std::env::var("CARGO_PKG_VERSION_MINOR").unwrap(),
          std::env::var("CARGO_PKG_VERSION_PATCH").unwrap(),
          std::env::var("CARGO_PKG_VERSION_PRE").unwrap(),
      )).unwrap();

      embed_resource::compile("resources/resources.rc", &[&out_dir]);
  }
  ```

**Comment by nabijaczleweli** [comment]
Does current `master` branch (at least a04a0058ba2a0a15fe58253e453c99707a5cf78f) work for you? It just always adds `$OUT_DIR` to the include path, which I think is moderately preferable for 99% of use-cases, and I'm not too too hot on adding another overload to `compile()` or parsing an environment variable, until that's actually needed, at least.

**Comment by roblabla** [comment]
Yeah, that's probably enough for my use-case. Thanks!

**Comment by nabijaczleweli** [comment]
Cool, released in [`v1.6.0`](https://github.com/nabijaczleweli/rust-embed-resource/releases/tag/v1.6.0).

**Comment by roblabla** [comment]
Can confirm it works!
